internal/application/utils/utils.go

```diff
@@ -30,3 +33,17 @@ func IsAllowedGrantType(oauthApplication *model.OAuthApplication, grantType stri
 	}
 	return false
 }
+
+// IsValidRedirectURI checks if the redirect URI is valid for the given OAuth application.
+func IsValidRedirectURI(oauthApplication *model.OAuthApplication, redirectURI string) bool {
+
+	// TODO: Provide support for callback uris with regex patterns.
```

Comment:
This is not required.

Reply:
Addressed with a4eb4127b25ebc4931241ef3491eb509782e5d0b